source/_patterns/01-molecules/navigation/login-control.mustache

```diff
@@ -0,0 +1,31 @@
+
+<div class="login-control"
+
+  {{#isLoggedIn}}
+
+     data-link-field-roots="{{linkFieldRoots}}"
+     {{{linkFieldData}}}
+     data-default-uri="{{defaultUri}}"
+     data-display-name="{{displayName}}"
+
+     {{#icon}}
+       data-icon-path-root="{{pathRoot}}"
+       data-icon-srcset="{{srcset}}"
+       data-icon-alt-text="{{altText}}"
+     {{/icon}}
```

Comment:
In Carouseltem we embed the `<picture>` (https://github.com/elifesciences/pattern-library/blob/develop/source/_patterns/01-molecules/components/carousel-item.mustache#L3-L19)

Reply:
I've never been totally happy with that approach, but it's the carousel, so let it go. In this case I think it's worth trying to get the data provided by journal and used by pattern-library (have always got the Carouseltem approach to fall back on if we need to).

Reply:
I'm not that big a fan either, but it does allow us to use enable things like WebP (through `<source>`).

Reply:
Good point. Okay, will go with the CarouselItem approach and see how it plays.